monkey/infection_monkey/network/network_scanner.py

```diff
@@ -6,7 +6,7 @@
 from infection_monkey.model.victim_host_generator import VictimHostGenerator
 from infection_monkey.network.info import local_ips, get_interfaces_ranges
 from infection_monkey.network import TcpScanner, PingScanner
-from infection_monkey.utils import is_windows_os
```

Comment:
Why did this change in this PR?

Reply:
Should have been a different branch, probably committed by mistake. Removed